src/js/components/footer.js

```diff
@@ -0,0 +1,41 @@
+import { newText, newMultimedia, newLink, newContainer } from './helpers';
+
+const createlinks = (datalinks) =>{
+  const arraylinks = datalinks.map((link)=>{
```

Comment:
This is a suggestion for the arrow function declaration: 

```
const arraylinks = datalinks.map((link) => {
...
}
```

Reply:
Done!. Add the space in the arrow on both functions